### DISCUSSION TITLE
Custom Metrics Charts

### DISCUSSION THREAD

**Discussion by MohmedMonsef** [locustio/locust#2984] [Q&A]
Is there a way to add a custom chart to the web UI.
The UI provides charts for number of users, response time and total request per second. 
But I need to add charts for other custom metrics, how to do that?

**Comment by andrewbaldwin44** [comment]
Hey @MohmedMonsef, sorry for the slow response. Custom charts are not officially supported within locust, however custom tables and reports are https://docs.locust.io/en/stable/extending-locust.html#extending-web-ui
